build.gradle

```diff
@@ -102,9 +101,12 @@ ext {
 repositories {
     // Main Maven repo
     mavenCentral()
-    // MovingBlocks Artifactory instance for libs not readily available elsewhere plus our own libs
+    // MovingBlocks Artifactory instance(s) for libs not readily available elsewhere plus our own libs
+    // TODO figure out why we need two URLs here
```

Comment:
Good question! We indeed shouldn't, since `libs-release-local` is indeed covered by the `virtual-repo-live` entry in Artifactory. So that is odd.

Reply:
Giving it a second try turns out to be working fine. Must have been some other quirk unrelated to the actual artifactory URL...